include/seqan3/alphabet/nucleotide/concept.hpp

```diff
@@ -68,11 +68,17 @@ constexpr bool is_nucleotide_v = is_nucleotide<type>::value;
 
 namespace seqan3
 {
-
-//!\brief A concept that indicates whether an alphabet represents nucleotides.
-//!\ingroup nucleotide
-//!\details Refines the seqan3::alphabet_concept.
+/*!\interface seqan3::nucleotide_concept <>
+ * \extends seqan3::alphabet_concept
+ * \brief A concept that indicates whether an alphabet represents nucleotides.
```

Comment:
... (e.g. `dna` or `dna5`). with links to the classes.
(I find it helpful, as often done in cppreference, to always include one example that satisfies this concept)
or is this maybe directly visible in doxygen? (..implemented by..)